app/javascript/react/components/GiraffeNewForm.js

```diff
@@ -1,10 +1,12 @@
 import React, { useState } from "react";
```

Comment:
Not a huge deal but looks like the additions weren't formatted with Prettier

Reply:
formatted